pkg/plugin/restore_pvc_action_plugin.go

```diff
@@ -144,3 +171,45 @@ func (p *NewPVCRestoreItemAction) Execute(input *velero.RestoreItemActionExecute
 	}, nil
 }
 
+func (p *NewPVCRestoreItemAction) createPod(item runtime.Unstructured) (*velero.RestoreItemActionExecuteOutput, error) {
+	var pod corev1.Pod
+	if err := runtime.DefaultUnstructuredConverter.FromUnstructured(item.UnstructuredContent(), &pod); err != nil {
+		return nil, errors.WithStack(err)
+	}
+
+	if metav1.HasAnnotation(pod.ObjectMeta, "vmware-system-vm-uuid") {
+		newPod := &corev1.Pod{
```

Comment:
Won't this remove all of the annotations?  I think we should only remove the bad ones.  Users will probably complain if their annotations get removed on restore.

Reply:
Fixed.